C/c4BlobStore.cc

```diff
@@ -34,21 +34,21 @@ namespace C4Blob {
 }
 
 
-
 #pragma mark - C4BLOBKEY:
 
 
 static constexpr slice
-    kBlobDigestStringPrefix = "sha1-",  // prefix of ASCII form of blob key ("digest" property)
-    kBlobFilenameSuffix = ".blob";      // suffix of blob files in the store
+        kBlobDigestStringPrefix = "sha1-",  // prefix of ASCII form of blob key ("digest" property)
```

Comment:
The alignment of the comments got lost here.

Reply:
Move to Clang-Format has fixed the alignment of the comments, although it has aligned the variables weirdly. A small thing, that I'm not sure how to fix, but I can investigate if needed.